ts/components/conversation/Emojify.tsx

```diff
@@ -41,5 +41,5 @@ export const Emojify = (props: Props): JSX.Element => {
     default:
   }
 
-  return <span style={{ fontSize: `${size}em`, userSelect: 'inherit' }}>{rendered}</span>;
+  return <span style={{ fontSize: `${size}rem`, userSelect: 'inherit' }}>{rendered}</span>;
```

Comment:
shouldn't this one actually be based on the font size (and so em)?

Reply:
resolved